### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by muja** [Saoneth/plex-custom-audio#14]
Hey @Saoneth great work! I'd love to use this but not sure if it still works since Plex is in active development and this hasn't been updated in 3 years. Seeing as it seemingly interacts with the Plex DB I don't want to break anything. Do you happen to know if this still works today?

**Comment by WkdXeqtr** [comment]
No, not on a current Plex. I tried it on 1.43.3 and it breaks at one specific point: Plex changed media_streams.extra_data from URL encoding to JSON and now reads that column with the ->> operator. The rows this writes are still in the old format, so the query that builds the track list dies with malformed JSON. The track does show up in the list, but selecting it returns HTTP 500 and it never plays.

Your library is not at risk though, if that is the worry. It only adds rows to media_streams and does not touch metadata or your files.

I needed it working, so I ended up writing my own version: https://github.com/WkdXeqtr/plex-external-audio

Separate implementation rather than a fork, Windows only for now, with an installer and a tray icon. The idea and the approach are Saoneth's.
